autoload/verilog_systemverilog.vim

```diff
@@ -587,6 +587,11 @@ function! verilog_systemverilog#VerilogErrorFormat(...)
   if (l:tool == "ncverilog")
     " Based on https://github.com/vhda/verilog_systemverilog.vim/issues/88
     set errorformat=%.%#:\ \*%t\\,%.%#\ \(%f\\,%l\|%c\):%m
+    set errorformat+=%.%#\ *%t\\,%.%#(%f\\,%l\|%c):\ %m
```

Comment:
Quite possibly, but be sure to take care of the colon after the first %# - the following formats can both happen:
ncsim: *E,SOMECADENCEERRORCODE: Message
ncsim: *E,ASRTST (filename.sv,123|12): Reason